src/main/java/io/cdap/e2e/utils/StorageClient.java

```diff
@@ -40,7 +45,7 @@ public class StorageClient {
 
   private static Storage getStorageService() throws IOException {
     return (null == storageService) ? StorageOptions.newBuilder().setProjectId(
-      PluginPropertyUtils.pluginProp(ConstantsUtil.PROJECT_ID)).build().getService() : storageService;
+        PluginPropertyUtils.pluginProp(ConstantsUtil.PROJECT_ID)).build().getService() : storageService;
```

Comment:
new line is seeen here too

Reply:
Changes has been done , as per the review comment